prehook/richcontainer/systemd_launcher_linux.go

```diff
@@ -43,6 +45,29 @@ func (l *systemdLauncher) Launch(opt *prehook.HookOptions, spec *specs.Spec) err
 
 	cmd := spec.Process.Args
 
+	if len(cmd) == 0 {
+		return errors.New("not set cmd")
+	}
+
+	//find PATH in env
+	sys_paths := []string{}
```

Comment:
Could you make the variable naming rule like other variables?
like local variables using camel writing ways with first letter lowercase.